guava-shaded/pom.xml

```diff
@@ -78,6 +74,8 @@
                 <includes>
                   <include>org.apache.cassandra:java-driver-guava-shaded</include>
                   <include>com.google.guava:guava</include>
+                  <include>com.google.guava:failureaccess</include>
+                  <include>com.google.j2objc:j2objc-annotations</include>
```

Comment:
Required because `SettableFuture` extends `AbstractFuture` where the `@ReflectionSupport(value = ReflectionSupport.Level.FULL)` annotation is present.